test/integration/nighthawk_grpc_service.py

```diff
@@ -60,12 +73,21 @@ def waitUntilServerListening(self):
                   self.server_ip, self.server_port)
     return False
 
+  """
+  Starts the Nighthawk gRPC service. Returns True upon success, after which the server_port attribute
+  can be queried to get the listening port.
+  """
+
   def start(self):
-    self.server_thread.daemon = True
-    self.server_thread.start()
-    return self.waitUntilServerListening()
+    self._server_thread.daemon = True
```

Comment:
(nit) In Python we typically add doc comments after the opening of the scope we are commenting on. To comment on a method we would do:

```python
def start(self):
  """
  Starts the Nighthawk gRPC service. Returns True upon success, after which the server_port
  attribute can be queried to get the listening port.
  """
```

Applies here and below.

Reply:
Whoops, fixed in 75ed6acc89f5937e90bc9ef617c7e1b535d1e0ac